libs/framework/include/celix_bundle_context.h

```diff
@@ -1311,7 +1311,7 @@ void celix_bundleContext_vlog(
  * @param defaultValue The default value if the property is not found.
  * @return The property value or the default value if the property is not found.
  */
-const char* celix_bundleContext_getProperty(celix_bundle_context_t *ctx, const char *key, const char *defaultVal);
+const char* celix_bundleContext_getProperty(celix_bundle_context_t *ctx, const char* key, const char* defaultVal);
```

Comment:
`const char* key` rather than `const char *key`,   `const celix_bundle_context_t *ctx` rather than `const celix_bundle_context_t* ctx`. 

Is there any coding convention involved?

Reply:
There is no written down coding convection and to be honest, the style has changed over the years.

I agree that it is good to write this down and maybe create clang-tidy config - or something like that - for this, but nor for this PR.